Delivery_to_lockers.app-meta.xml

```diff
@@ -1,25 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why did you delete this file?

Reply:
I accidentally rushed and merged the pull request without waiting for the review.